apple/Handlers/RNHoverHandler.m

```diff
@@ -155,7 +155,8 @@ - (void)configure:(NSDictionary *)config
 - (RNGestureHandlerEventExtraData *)eventExtraData:(UIGestureRecognizer *)recognizer
 {
   return [RNGestureHandlerEventExtraData forPosition:[recognizer locationInView:recognizer.view]
-                                withAbsolutePosition:[recognizer locationInView:recognizer.view.window]];
+                                withAbsolutePosition:[recognizer locationInView:recognizer.view.window]
+                                     withPointerType:UITouchTypePencil];
```

Comment:
Hover also works with mouse

Reply:
Currently we don't have option to distinguish between `mouse` and `apple pen` on iOS. I suggest leaving is as it is and describing this behavior in docs.